hamcrest/Hamcrest/Core/HasToString.php

```diff
@@ -45,7 +45,7 @@ protected function featureValueOf($actual)
     }
 
     /**
-     * Does array size satisfy a given matcher?
+     * Creates a matcher that matches any examined object whose toString method returns a value equalTo the specified string.
```

Comment:
Please use text from the [relevant java file](https://github.com/hamcrest/JavaHamcrest/blob/master/hamcrest/src/main/java/org/hamcrest/object/HasToString.java) to preserve formatting.

In this case only the `<code>...</code>` text was lost, but I'm not sure if PhpDoc supports it at all.